packages/http-request-handler/src/createSimpleServer.helper.ts

```diff
@@ -0,0 +1,63 @@
+import http from 'http';
+import url from 'url';
+import { AddressInfo } from 'net';
+import { HttpMethod } from '@coolio/http';
+
+export interface SimpleEndpoint {
+  method: HttpMethod;
+  route: RegExp | string;
+  response: string | ((req: http.IncomingMessage) => string);
+}
+
+export interface CreateSimpleServerParams {
+  status: number;
+  headers: http.OutgoingHttpHeaders;
+  endpoints: SimpleEndpoint[];
+}
+
+export interface SimpleServer {
+  port: number;
+  host: string;
+  fullAddress: string;
+  close: () => Promise<void>;
+}
+
+export const createSimpleServer = ({
```

Comment:
Can you add a `// TODO` for me to just use `express` for testing in the future? 😅